### ISSUE TITLE
reading of GFS history files fails (on reading 'time_iso')

### ISSUE THREAD

**Issue by jswhit** [NOAA-EMC/GSI#348]
https://github.com/ufs-community/ufs-weather-model/issues/1052 added a new 'time_iso' character array variable in the GFS history files.  The ncio module that GSI uses to read the history files chokes when trying to read this.  A fix is at https://github.com/NOAA-EMC/NCEPLIBS-ncio/pull/53.

https://github.com/NOAA-EMC/GSI/pull/340 and updating the hpc-stack with an updated NCEPLIBS-ncio with the above fix will be necessary for GSI to work with ufs-weather-model/develop.

**Comment by RussTreadon-NOAA** [comment]
@MichaelLueken-NOAA and @WalterKolczynski-NOAA , is this the issue we must resolve before we can restore DA cycling in the global workflow?   Following some of the issue links above suggests that we are waiting for hpc-stack issue [#437](https://github.com/NOAA-EMC/hpc-stack/issues/437).   

We have issues referencing issues.   It would be could to list somewhere, perhaps g-w issue [#711](https://github.com/NOAA-EMC/global-workflow/issues/711) all the open issues or PRs which need to be closed in order to move forward with #711.

**Comment by WalterKolczynski-NOAA** [comment]
Yes, this is what I'm waiting on to fix cycling in global workflow.

**Comment by arunchawla-NOAA** [comment]
I am trying to follow this and am wondering why this issue is still open ? Did the update to ncio module 1.1.2 not fix the problem ?

**Comment by MichaelLueken** [comment]
@arunchawla-NOAA The ncio/1.1.2 fixes the issue.  The problem is that the ncio/1.1.2 needs to be available on WCOSS2 before the module can be used in the GSI.  All library modules are defined in modulefiles/gsi_common.lua.  While replacing ncio/1.0.0 with ncio/1.1.2 will work for all machines except for WCOSS2, there is no point in applying a change to the authoritative repository that will cause the GSI to fail to compile on that machine.  Once ncio/1.1.2 has been added to WCOSS2, the update to the ncio version can be made to modulefiles/gsi_common.lua.

**Comment by MichaelLueken** [comment]
@arunchawla-NOAA Following what you said in issue [#711](https://github.com/NOAA-EMC/global-workflow/issues/711) in global-workflow, since ncio/1.1.2 fixes the ability to read GFS history files and allows the GSI to run while reading and writing NetCDF files, I will go ahead and close this issue and open a new issue to bring ncio/1.1.2 into modulefiles/gsi_common.lua once it is available on WCOSS2.

**Comment by MichaelLueken** [comment]
Thanks to the modifications made by @jswhit, ncio/1.1.2 is able to both properly read and write time_iso, as well as one dimensional arrays that were causing issues following the original update to read time_iso, this issue has been corrected.  A new issue will now be opened detailing when this updated version is available on WCOSS2 and replacing ncio/1.0.0 with ncio/1.1.2 in modulefiles/gsi_common.lua.
